app/models/jobsModel.js

```diff
@@ -0,0 +1,14 @@
+const db = require('../database/dbConfig.js')
+
+async function insertJob(job){
```

Comment:
It's also good to write a couple lines of comment as javadoc comment just to define the parameter and how the json shoule look like